client/package.json

```diff
@@ -9,7 +9,7 @@
     "mongojs": "~0.18.2",
     "request": "^2.57.0",
     "socket.io": "~0.9.14",
-    "t2": "git://github.com/tessel/t2-cli.git#jh-rig-test",
+    "t2": "git://github.com/tessel/t2-cli.git#jon-rig-improvements",
```

Comment:
This should point to a solid released version of the CLI (probably 0.0.11) by the time this is merged.